ra/ra.go

```diff
@@ -1224,7 +1231,7 @@ func (ra *RegistrationAuthorityImpl) issueCertificateOuter(
 	defer ra.inflightFinalizes.Dec()
 
 	// Step 3: Issue the Certificate
-	cert, err := ra.issueCertificateInner(
+	cert, profileName, profileHash, err := ra.issueCertificateInner(
```

Comment:
Ah, I missed that you'd fixed this for issuePrecertificateInner but not for issueCertificateInner. Let's have this also return a *certProfileWithID rather than having two extra return values. This also addresses Jacob's comment about the return values not being self-documenting.

Reply:
🤦🏼‍♂️

Reply:
`certProfileWithID` is not currently exported from the CA, but I guess could be. The RA's `issueCertificateInner` would then return `(*x509.Certificate, *ca.CertProfileWithID, error)`. That seems a bit odd to me. What do you think?

Reply:
Alternatively, and is probably what I'll do, I could just redefine it in the RA.